migrations/sql/V2021.06.14.11.44__update_camp_application_activity.sql

```diff
@@ -0,0 +1,43 @@
+CREATE TABLE IF NOT EXISTS activity_summary_building_detail_xref (
+	activity_summary_id INTEGER NOT NULL REFERENCES activity_summary(activity_summary_id), 
+	activity_detail_id INTEGER NOT NULL REFERENCES activity_detail(activity_detail_id),
+	
+    PRIMARY KEY(activity_summary_id, activity_detail_id)
+);
+ALTER TABLE activity_summary_building_detail_xref OWNER TO mds;
+
+CREATE TABLE IF NOT EXISTS activity_summary_staging_area_detail_xref (
+	activity_summary_id INTEGER NOT NULL REFERENCES activity_summary(activity_summary_id), 
+	activity_detail_id INTEGER NOT NULL REFERENCES activity_detail(activity_detail_id),
+	
+    PRIMARY KEY(activity_summary_id, activity_detail_id)
+);
+ALTER TABLE activity_summary_staging_area_detail_xref OWNER TO mds;
+
+
+ALTER TABLE camp DROP COLUMN camp_name;
+ALTER TABLE camp DROP COLUMN camp_number_people;
+ALTER TABLE camp DROP COLUMN camp_number_structures;
+
```

Comment:
These values were saved to the camp_summary, but were never used. They should have existed on the camp_detail.